qadi_multi_perspective.py

```
@@ -0,0 +1,284 @@
+    
+    emoji = emoji_map.get(pr.perspective, "📋")
+    header = f"{emoji} {pr.perspective.value.title()} Perspective"
+    if is_primary:
+        header += " (Primary)"
+    
+    print(f"\n{'=' * 70}")
+    print(f"{header}")
+    print(f"{'=' * 70}")
+    
+    # Core Question
+    print(f"\n**Core Question:**")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)